frontend/packages/integration-tests-cypress/tests/app/demo-dynamic-plugin.spec.ts

```diff
@@ -0,0 +1,184 @@
+    });
+  if (isLocalDevEnvironment()) {
+    // for local dev env just trigger any change in the return value to activate the toast
+    cy.intercept('/api/check-updates*', { plugins: [] }).as('checkUpdates');
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.wait(CHECK_UPDATE_WAIT);
+    cy.wait('@checkUpdates');
+    cy.log('Running plugin test locally using bridge.');
+  } else {
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.wait(CHECK_UPDATE_WAIT);
+    cy.log(`Running plugin test on ci using PLUGIN_PULL_SPEC: ${PLUGIN_PULL_SPEC}.`);
+  }
+  cy.get(warningToast)
+    .should('exist')
+    .then(() => {
+      cy.reload();
+      if (!enable) {
+        cy.byLegacyTestID(PLUGIN_NAME).click();
+        detailsPage.titleShouldContain(PLUGIN_NAME);
+        detailsPage.clickPageActionFromDropdown('Delete ConsolePlugin');
+        modal.shouldBeOpened();
+        modal.submit();
+      }
+    });
+};
+
+const dynamicNavTest = (navID: string) => {
+  nav.sidenav.clickNavLink(['Demo Plugin', `Dynamic Nav ${navID}`]);
+  cy.get('.pf-c-title.pf-m-2xl').should('contain', `Dynamic Page ${navID}`);
+  cy.get(infoMessage).should('exist');
+  cy.get(warningMessage).should('exist');
+  cy.get(hintMessage).should('exist');
+  cy.get('.pf-c-card').should('have.length', 50);
+};
+
+describe('Demo dynamic plugin test', () => {
+  before(() => {
+    cy.login();
+    nav.sidenav.switcher.changePerspectiveTo('Administrator');
+    nav.sidenav.switcher.shouldHaveText('Administrator');
+    cy.visit('/k8s/all-namespaces/import');
+    cy.readFile(`${PLUGIN_PATH}/oc-manifest.yaml`).then((textManifest) => {
+      const yamlManifest = safeLoadAll(textManifest);
+      const yamlManifestUpdatedPullSpec = yamlManifest.map((yamlDefinition) => {
+        if (yamlDefinition.kind === 'Deployment' && !isLocalDevEnvironment() && PLUGIN_PULL_SPEC) {
+          return _.defaultsDeep(
+            {},
+            {
+              spec: {
+                template: {
+                  spec: {
+                    containers: [{ name: 'console-demo-plugin', image: PLUGIN_PULL_SPEC }],
+                  },
+                },
+              },
+            },
+            yamlDefinition,
+          );
+        }
+        return yamlDefinition;
+      });
+      const finalYAMLText = yamlManifestUpdatedPullSpec.map((obj) => safeDump(obj)).join('---\n');
+      yamlEditor.isImportLoaded();
+      yamlEditor.setEditorContent(finalYAMLText).then(() => {
+        yamlEditor.clickSaveCreateButton();
+        cy.get(errorMessage).should('not.exist');
+        cy.byTestID('success-icon').should('have.length', 5);
+        cy.byTestID('import-more-yaml').should('exist');
+        enableDemoPlugin(true);
+      });
+    });
+  });
+
+  after(() => {
+    enableDemoPlugin(false);
+    cy.deleteProject(PLUGIN_NAME);
+    cy.logout();
+  });
+
+  afterEach(() => {
```

Comment:
.nit `after[all]` is typically after `afterEach`

Reply:
done